src/ralph/backends/conf.py

```diff
@@ -0,0 +1,91 @@
+"""Configurations for Ralph backends."""
+
+from pydantic import BaseModel, BaseSettings
+
+from ralph.backends.data.clickhouse import ClickHouseDataBackendSettings
+from ralph.backends.data.es import ESDataBackendSettings
+from ralph.backends.data.fs import FSDataBackendSettings
+from ralph.backends.data.ldp import LDPDataBackendSettings
+from ralph.backends.data.mongo import MongoDataBackendSettings
+from ralph.backends.data.s3 import S3DataBackendSettings
+from ralph.backends.data.swift import SwiftDataBackendSettings
+from ralph.backends.http.async_lrs import LRSHTTPBackendSettings
+from ralph.backends.lrs.clickhouse import ClickHouseLRSBackendSettings
+from ralph.backends.lrs.fs import FSLRSBackendSettings
+from ralph.backends.stream.ws import WSStreamBackendSettings
```

Comment:
Aren't some backend dependencies optional?
Thus, if a backend dependency is missing, does importing it still work?
Could we load them dynamically?
For example, we could get a list of data backend module names with:
```python
map(
    lambda x: f"ralph.backends.data.{x[1]}",
    pkgutil.iter_modules(importlib.util.find_spec("ralph.backends.data").submodule_search_locations)
)
```

Reply:
Good catch!

Reply:
Opened issue https://github.com/openfun/ralph/issues/458 as it will require a bit of work!